Dockerfile

```diff
@@ -1,22 +1,45 @@
-FROM python:3.7-slim-stretch
+FROM node:current-alpine AS NODEBUILD
+# copy directory to /opt and work inside it
+WORKDIR /opt
+COPY . /opt
+# run the npm portion of build-static script
+RUN rm -rf static && mkdir -p static/
+RUN npm install -g npm && \
+    npm install && \
+    npm run build
 
-# Install packages required for uswgi
-RUN apt-get update
-RUN apt-get -y install build-essential
-RUN apt-get -y install mime-support
 
-# Install requirements and uwsgi server for running python web apps
+FROM python:3.9-slim AS PYTHONBUILD
+# install missing system deps
+RUN apt-get update && apt-get -y install build-essential mime-support libpcre3-dev libsass-dev mime-support
```

Comment:
mime-support is listed twice.

Reply:
Whoops. I was reducing intermediate build steps and didn't catch this one.